### ISSUE TITLE
Hyperparameters exposure

### ISSUE THREAD

**Issue by gcattan** [pyRiemann/pyRiemann-qiskit#13]
Ok. Like `SVC`, `QuanticSVM` should allow to set SVM hyper-parameters in its `init`.

_Originally posted by @qbarthelemy in https://github.com/pyRiemann/pyRiemann-qiskit/pull/2#discussion_r734956187_

**Comment by gcattan** [comment]
Todo list:
- gamma (SklearnSVM)
- feature_map (QSVM and VQC)
- number of shots (QSVM and VQC)
- optimizer (VQC)
- variational form (VQC)
- enforce_spd (QSVM and VQC)
- number of iteration (QSVM, VQC, SklearnSVM)
- output normalization  (QSVM, VQC, SklearnSVM)
- L2 regularisation  (QSVM, VQC, SklearnSVM)

**Comment by qbarthelemy** [comment]
Can we close this issue?

**Comment by gcattan** [comment]
Hello, 

I was not sure how to manage stories.
There are four parameters to expose again, but I can create follow-up issues and reference them in this one if you prefer.

**Comment by gcattan** [comment]
After double-checking, it seems that these last parameters are not accessible directly in the version of qiskit we used. 
I think we can safely close this issue and add follow-up for C and max_iter parameters, after #15.

**Comment by qbarthelemy** [comment]
Ok.
